docs/process/metrics.md

```diff
@@ -0,0 +1,51 @@
+# GitHub Desktop Metrics
+Desktop is a free, open source product that is maintained and worked on by the small, remote team of @desktop/core, @desktop/comrades, and the general open-source community. As a result, we do not have the resources to do continuous, detailed user studies of GitHub Desktop users and usage. In other words, we love you, we want to build the right features to help you, and yet we don't know enough about you as a whole to make informed prioritization decisions. 
+
+We measure certain events in the GitHub Desktop app in order to learn more about how you are using the app. This helps us prioritize features and bugs and understand what things are most important to the largest number of people. Our understanding of these metrics is supplemented by some user research, usability testing, and feedback from the community. This document is intended to provide information about each thing we're measuring and what we're using it for.
+
+You can always see our recently released user facing features and fixes in the [release notes](https://desktop.github.com/release-notes/) of our marketing page.
+
+## Dimensions
+These are general metrics about users that are aggregated to understand general user behavior and to help us segment usage patterns. These helps us prioritize work, predict technical and performance implications of our features, and plan for devOps needs. 
+
+| Metric | Description | Justification |
```

Comment:
Perhaps add a “type” or “example” column for more clarity?

Reply:
If I add a `type` or `example` column`, what would you expect to be in each row?